tests/template_tests/test_context.py

```diff
@@ -182,6 +184,26 @@ def test_copy_request_context_twice(self):
         """
         RequestContext(HttpRequest()).new().new()
 
+    @ignore_warnings(category=RemovedInDjango20Warning)
+    def test_has_key(self):
+        a = Context({'a': 1})
+        b = RequestContext(HttpRequest(), {'a': 1})
+        msg = "Context.has_key() is deprecated in favor of the 'in' operator."
+        msg2 = "RequestContext.has_key() is deprecated in favor of the 'in' operator."
+
+        with warnings.catch_warnings(record=True) as warns:
+            warnings.simplefilter('always')
+            self.assertEqual(a.has_key('a'), True)
+            self.assertEqual(a.has_key('b'), False)
+            self.assertEqual(b.has_key('a'), True)
+            self.assertEqual(b.has_key('b'), False)
```

Comment:
I thought it's safer since `assertTrue`/`False` will pass for any truthy/falsey value.